gobblin-utility/src/main/java/gobblin/util/concurrent/TaskSchedulerFactory.java

```diff
@@ -0,0 +1,54 @@
+package gobblin.util.concurrent;
+
+import com.google.common.base.Optional;
+import com.google.common.collect.Maps;
+import org.apache.commons.lang.StringUtils;
+
+import java.util.Map;
+
+public class TaskSchedulerFactory {
```

Comment:
This is so the TaskSchedulers can be loaded by name.  I want to be able to allow the TaskScheduler to be configured in the gobblin config so that we can switch between them before we are certain that the new Scheduler is performing correctly.